### ISSUE TITLE
Update regarding Sending SMS using Pinpoint

### ISSUE THREAD

**Issue by ShivaniMehendarge** [aws-samples/retail-demo-store#207]
For SMS using Pinpoint, RDS Campaigns and workshop is setup to use a dedicated long code that the users can obtain using AWS pinpoint console. Starting **Feb 15**, Pinpoint has introduced some **changes to the process of registering a long code with campaigns in Pinpoint.**  -[1]

With the new process, it is first required to register one's company and create a 10DLC (10-digit long code) Campaign using the console. After this, the campaign is sent to a 3rd party campaign registry, which approves or rejects the campaign based on the information provided. Some campaigns might take a week or longer for approval. This will affect the current setup in Retail Demo Store and will require some changes to the workshop and campaign used for sending SMS to the users.

Resolution Steps:
- Update the instructions to get a long code in the Pinpoint Workshop
- Update the CFN template parameter description
- Update campaigns to follow the registering process (this might lead to more changes; might need to verify if the auto deployment works even with unregistered campaign

-[1] The information about the change can be found here: https://docs.aws.amazon.com/pinpoint/latest/userguide/settings-10dlc.html

**Comment by changux** [comment]
Hi people. To avoid the 10DLC, it is possible to register a short-code toll-free number? The restriction (3 messages x min) is ok for testing purposes.

**Comment by damiendaemon** [comment]
@changux Yes it is OK to register a toll-free number in the US and it works fine in my testing - you do not need a short-code. See PR https://github.com/aws-samples/retail-demo-store/pull/255 for updated instructions on this.

PR https://github.com/aws-samples/retail-demo-store/pull/251 also enables you to send messages to toll-free numbers that you registered with AWS (but not the same number that you are sending from).
